### PR TITLE
feat(caluma)!: remove caluma service

### PR DESCRIPTION
**Pull request by open-dynaMIX**
Caluma is now fully integrated in camac-ng. There si no need for a
caluma service anymore.

This commit removes the caluma service related config and updates the
variables that have to be set for camag-ng.

### CODE REVIEW

**Comment by anehx** [comment]
Let's wait with this until the prod release on moday is through, I'll review then

File: defaults/main.yml [hunk 1]
```diff
@@ -288,110 +288,30 @@ camac_portal_uri: "http://caluma.127.0.0.1.nip.io"
...
 
 # Caluma language settings
-caluma_language_code: "de"
-caluma_languages: "de,fr"
+camac_caluma_language_code: "de"
+camac_caluma_languages: "de,fr"
 
 # Enable caluma historical API
-caluma_enable_historical_api: true
+camac_enable_historical_api: true
```
Comment (czosel) [inline defaults/main.yml]:
```suggestion
camac_caluma_enable_historical_api: true
```

**Review by czosel — APPROVED** [review]
I was able to deploy this with one minor change:

File: defaults/main.yml (hunk 1 above)
Comment (open-dynaMIX) [reply defaults/main.yml]: True, also for consistency, this should be called `camac_caluma_enable_historical_api`.

Comment (czosel) [reply defaults/main.yml]: Yeah, that's my suggested change :wink:

Comment (open-dynaMIX) [reply defaults/main.yml]: Haha, true. For my defense: I was on mobile ;)

**Pull request merged**